src/features/device-types/storage/s3.ts

```diff
@@ -57,6 +57,17 @@ function createS3Client() {
 
 const s3Client = createS3Client();
 
+function logUnauthenticatedWarning(clientS3, pathS3, err): void {
```

Comment:
```suggestion
function logUnauthenticatedWarning(clientS3: UnauthenticatedS3Facade | AWSWrapper.AWS.S3, pathS3: string, err: any): void {
```

Missing argument types